### PR TITLE
EE-1152: Updated BCs and DCs so Postgres is seperate from Metabase

### PR DESCRIPTION
**Pull request by danieltruong**

### CODE REVIEW

**Review by mtCarto — APPROVED** [review]
Looks good, nitpick: the postgres dc could be part of the metabase dc's objects array, rather than fully split out on it's own.

File: openshift/templates/metabase/metabase-postgres.dc.json [hunk 1]
```diff
@@ -0,0 +1,268 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "metabase-postgres-dc"
```
Comment (MaxWardle) [inline openshift/templates/metabase/metabase-postgres.dc.json:5]: Isn't there a metabase-epic as well? do we use the same template for that DC? If not we should include that template in this folder or change this template name to just metabase-dc or something like that I think.

**Review by MaxWardle — CHANGES_REQUESTED** [review]

File: openshift/templates/metabase/metabase-postgres.dc.json (hunk 1 above)
Comment (mtCarto) [reply openshift/templates/metabase/metabase-postgres.dc.json:5]: yeah that's along the lines of my comment, 1 template with both in configs in it versus 2 separate files

Comment (MaxWardle) [reply openshift/templates/metabase/metabase-postgres.dc.json:5]: ah, I thought the whole metabase-dc file was getting deleted. I blame pre-coffee brain... Would be nice for these to be combined into one file though for easier deployment.  Would like to see a "role" label on the other template as well for the network security policies like this template does :)

File: openshift/templates/metabase/metabase-postgres.dc.json [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+          "name": "${NAME_DATABASE}",
+          "app": "${NAME_DATABASE}"
+        },
+        "template": {
+          "metadata": {
+            "name": "${NAME_DATABASE}",
+            "labels": {
+              "name": "${NAME_DATABASE}",
+              "app": "${NAME_DATABASE}",
+              "role": "${NAME_DATABASE}-${ENVIRONMENT_NAME}"
```
Comment (MaxWardle) [inline openshift/templates/metabase/metabase-postgres.dc.json:51]: this one 👆

**Pull request closed without merging**